server/routes/user.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+var userAPI = require('./../api/controllers/users');
+
+var user = function(app) {
+
+  app.post('/user/create', function(req, res) {
+    var thisUser = userAPI.createUserObj(req);
+
+    var newUser = userAPI.createUser(thisUser, function(err, user) {
```

Comment:
I don't think we will need to get this variable. I also don't think the method returns anything but 'undefined'. Look at the changes in company.